Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * Yul: Raise warning for switch statements that only have a default and no other cases.
  * Output compilation error codes.
+ * Natspec: Add fields "kind" and "version" to the JSON output.
```

Comment:
```suggestion
 * NatSpec: Add fields "kind" and "version" to the JSON output.
```

For consistency with rest of changelog

Reply:
While this is true for the changelog in the code we never use that camel case spelling. Seems generally inconsistent

Reply:
The docs use the same "spelling" https://solidity.readthedocs.io/en/v0.6.9/natspec-format.html